pkg/ignition/builder.go

```diff
@@ -31,9 +31,10 @@ type ignitionBuilder struct {
 	noProxy                   string
 	hostname                  string
 	ironicAgentVlanInterfaces string
+	additionalNTPServers      string
 }
 
-func New(nmStateData, registriesConf []byte, ironicBaseURL, ironicInspectorBaseURL, ironicAgentImage, ironicAgentPullSecret, ironicRAMDiskSSHKey, ipOptions string, httpProxy, httpsProxy, noProxy string, hostname string, ironicAgentVlanInterfaces string) (*ignitionBuilder, error) {
+func New(nmStateData, registriesConf []byte, ironicBaseURL, ironicInspectorBaseURL, ironicAgentImage, ironicAgentPullSecret, ironicRAMDiskSSHKey, ipOptions string, httpProxy, httpsProxy, noProxy string, hostname string, ironicAgentVlanInterfaces string, additionalNTPServers string) (*ignitionBuilder, error) {
```

Comment:
I'd slightly prefer additionalNTPServers to be passed as []string and the split to be done at the caller side. Just for a cleaner API here (the Builder is used externally - in the assisted service).

Reply:
Done